python/ray/tune/tests/test_result_grid.py

```diff
@@ -224,6 +224,26 @@ def f(config):
     assert not any(key in representation for key in AUTO_RESULT_KEYS)
 
 
+def test_result_grid_repr(ray_start_2_cpus):
+    def f(config):
+        with tune.checkpoint_dir(step=0) as checkpoint_dir:
+            path = os.path.join(checkpoint_dir, "checkpoint")
+            with open(path, "w") as f:
+                f.write(json.dumps({"step": 0}))
+
+        from ray.air import session
+
+        session.report({"loss": 1})
```

Comment:
Also make sure the `Checkpoint` gets imported with `from ray.air import Checkpoint`

```suggestion
        session.report({"loss": 1}, checkpoint=Checkpoint.from_dict({"step": 0}))
```

Reply:
Yes, it has been imported at the beginning.